.github/workflows/maven.yml

```diff
@@ -232,13 +232,18 @@ jobs:
         uses: aquasecurity/trivy-action@master
         env:
           TRIVY_DOWNLOAD_JAVA_DB_ONLY: true
+          TRIVY_JAVA_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-java-db,ghcr.io/aquasecurity/trivy-java-db:1
         with:
           scan-type: image
+          timeout: 10m0s
           cache-dir: .trivy
 
       - name: Trivy Vulnerability Scan
         if: ${{ matrix.os == 'ubuntu-latest' }}
         uses: aquasecurity/trivy-action@master
+        env:
```

Comment:
Should also add `TRIVY_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-db,ghcr.io/aquasecurity/trivy-db` to the env here to add a mirror for their main DB as well

Reply:
If we download the DB in an explicit step, this shouldn;t be needed.